src/main/java/aichan/Storage.java

```diff
@@ -0,0 +1,86 @@
+package aichan;
+
+import aichan.task.Task;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Represents a storage which deals with loading tasks and saving tasks.
+ */
+public class Storage {
+    private final String filePath;
+
+    /**
+     * Constructs a storage object.
+     *
+     * @param filePath File path for storing and loading tasks.
+     */
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+    /**
+     * Saves the tasks to the file.
+     *
+     * @param tasks TaskList contains current tasks.
+     * @throws AiChanException If an error occurs when saving tasks.
+     */
+    public void save(TaskList tasks) throws AiChanException {

```

Comment:
Maybe can give a more descriptive function name?

```suggestion
    public void saveTasks(TaskList tasks) throws AiChanException {
```